### PR TITLE
Feat/#39521

### PR DESCRIPTION
**Pull request by SDmayank**

### CODE REVIEW

File: src/Server.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+       res.send( 'i am okk' );
+    });
+    }
+    run(): Server {
+        this.app.listen(this.config.port, ( err: any ): any => {
+            if ( err ) {
+                console.log(err);
+                throw err;
+            }
+            console.log(`server is running on ${this.config.port}`);
```
Comment (VisaaliSuccessive) [inline src/Server.ts]: print current env also

File: src/Server.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+     constructor (private config: Iconfig) {
+        this.app = express();
+    }
+    bootstrap() {
+        this.setupRoutes();
+        return this;
+    }
+    setupRoutes(): void {
+     this.app.get('/`health-check', ( req: express.Request, res: express.Response) => {
+       res.send( 'i am okk' );
```
Comment (VisaaliSuccessive) [inline src/Server.ts]: message must be written properly

File: src/Server.ts (hunk 1 above)
Comment (SDmayank) [reply src/Server.ts]: done

File: src/Server.ts (hunk 2 above)
Comment (SDmayank) [reply src/Server.ts]: done

**Review by VisaaliSuccessive — APPROVED** [review]

**Pull request merged**